pokedex_async_redux/lib/features/pokemon_details/widget/pokemon_ability.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:dartx/dartx.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:pokedex_async_redux/api/model/models.dart';
+import 'package:pokedex_async_redux/utilities/colors.dart';
+import 'package:pokedex_async_redux/utilities/constants.dart';
+import 'package:flutter/material.dart';
+
+class AbilityView extends StatelessWidget {
+  const AbilityView({
+    required this.abilities,
+    Key? key,
+  }) : super(key: key);
+
+  final List<PokemonAbilities>? abilities;
```

Comment:
haven't I explained before in terms of collection objects, as much as possible never resort to make them as nullable. Instead make them non-nullable then pass an empty list instead if ever 🙂